src/python/pants/init/engine_initializer.py

```diff
@@ -196,30 +198,36 @@ def __init__(self, invalid_goals):
       )
       self.invalid_goals = invalid_goals
 
-  def run_goal_rules(self, options_bootstrapper, options, goals, target_roots):
+  def run_goal_rules(
+    self,
+    options_bootstrapper: OptionsBootstrapper,
+    options: Options,
+    goals: Iterable[str],
+    target_roots: TargetRoots,
+  ):
     """Runs @goal_rules sequentially and interactively by requesting their implicit Goal products.
 
     For retryable failures, raises scheduler.ExecutionError.
 
-    :param list goals: The list of requested goal names as passed on the commandline.
-    :param TargetRoots target_roots: The targets root of the request.
+    :param goals: The list of requested goal names as passed on the commandline.
+    :param target_roots: The targets root of the request.
```

Comment:
`target_roots` param name is `targets root`? Should that be `target roots of the request` instead?

Reply:
Good catch! Will fix in the followup to this PR.